pkg/tfgen/docs_test.go

```diff
@@ -44,40 +44,48 @@ import (
 var accept = cmdutil.IsTruthy(os.Getenv("PULUMI_ACCEPT"))
 
 type testcase struct {
-	Input    string
-	Expected string
+	name   string
+	input  string
+	expect autogold.Value
```

Comment:
I think `autogold.ExpectFile(t, autogold.Raw(text))` might address the failures you were seeing on windows.